packages/pds/src/auth-verifier.ts

```diff
@@ -484,18 +481,38 @@ export class AuthVerifier {
         { audience: [this.dids.pds] },
       )
 
-      const { sub } = result.claims
+      const { sub, scope } = result.claims
       if (typeof sub !== 'string' || !sub.startsWith('did:')) {
         throw new InvalidRequestError('Malformed token', 'InvalidToken')
       }
 
+      if (!scope?.includes('transition:generic')) {
+        throw new AuthRequiredError('Missing required scope', 'InvalidToken')
```

Comment:
Folks will run into this on release, it would be nice if we can help them out a bit:
```suggestion
      if (!scope?.includes('transition:generic')) {
        throw new AuthRequiredError('Missing required scope: transition:generic', 'InvalidToken')
```